webvh/webvh/did/witness_manager.py

```diff
@@ -39,9 +39,11 @@ class WitnessManager:
     def __init__(self, profile: Profile):
         """Initialize the witness manager."""
         self.profile = profile
+        self.witness_alias_suffix = '@witness'
```

Comment:
I'd suggest having this as a module scoped constant instead of a class instance scoped variable.